base/package.mustache

```diff
@@ -11,7 +11,7 @@
     "coverage": "istanbul cover _mocha",
     "start": "node src/server",
     "test": "NODE_ENV=test mocha",
-    "watch": "mocha --watch",
+    "watch": "NODE_ENV=test mocha --watch",
```

Comment:
probably should ditch the whole `"watch"` thing in favour of running `npm test -- --watch`